package.json

```diff
@@ -31,6 +31,7 @@
     "github-slugger": "^1.3.0",
     "gray-matter": "^4.0.2",
     "html-react-parser": "^0.13.0",
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
Next.js has fetch build in, [see here.](https://nextjs.org/blog/next-9-4#improved-built-in-fetch-support)

Reply:
right, thanks!